examples/custom-rlpx-subprotocol/src/subprotocol/protocol/message.rs

```diff
@@ -0,0 +1,88 @@
+use reth_eth_wire::{protocol::Protocol, Capability};
```

Comment:
ideally we reference the rlpx spec here:

https://github.com/ethereum/devp2p/blob/master/rlpx.md

and mention that this is a simple ping pong protocol that also supports sending messages